drivers/vector/simulator.py

```diff
@@ -28,7 +28,11 @@ def expand_single_gate(gate, i, num_qbits):
 
 
 def expand_double_gate(gate, i, j, num_qbits):
-    assert i < j, "Operation only implemented for i < j"
+    if j < i:
```

Comment:
Same thing here, i==j was previously asserted not to happen, and is now left unhandled.